test/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishAProjectWithAllFeatures.cs

```diff
@@ -147,6 +147,97 @@ public static IEnumerable<object[]> PublishData
                         "NoneCopyOutputAlways.txt",
                         "NoneCopyOutputPreserveNewest.txt",
                         "CopyToOutputFromProjectReference.txt",
+                        "System.Spatial.dll",
+                        "System.AppContext.dll",
```

Comment:
Is this good?  It seems wrong to me that if I target `netcoreapp1.0` and publish, that I will get all these assemblies in my publish directory.